services/ui-src/src/components/cards/EntityCard/EntityCardBottomText.tsx

```diff
@@ -0,0 +1,96 @@
+// components
+import { Box, Flex, Text } from "@chakra-ui/react";
+// utils
+import { AnyObject, ModalDrawerEntityTypes } from "types";
+
+export const EntityCardBottomText = ({
+  entityType,
+  entityCompleted,
+  formattedEntityData,
+}: Props) => {
+  switch (entityType) {
+    case ModalDrawerEntityTypes.ACCESS_MEASURES:
+      return (
+        <>
+          {entityCompleted ? (
+            <>
+              <Flex sx={sx.highlightContainer}>
+                <Box sx={sx.highlightSection}>
+                  <Text sx={sx.subtitle}>Provider</Text>
+                  <Text sx={sx.subtext}>{formattedEntityData?.provider}</Text>
+                </Box>
+                <Box sx={sx.highlightSection}>
+                  <Text sx={sx.subtitle}>Region</Text>
+                  <Text sx={sx.subtext}>{formattedEntityData?.region}</Text>
+                </Box>
+                <Box sx={sx.highlightSection}>
+                  <Text sx={sx.subtitle}>Population</Text>
+                  <Text sx={sx.subtext}>{formattedEntityData?.population}</Text>
+                </Box>
+              </Flex>
+              <Text sx={sx.subtitle}>Monitoring Methods</Text>
+              <Text sx={sx.subtext}>
+                {formattedEntityData?.monitoringMethods.join(", ")}
+              </Text>
+              <Text sx={sx.subtitle}>Frequency of oversight methods</Text>
+              <Text sx={sx.subtext}>{formattedEntityData.methodFrequency}</Text>
+            </>
+          ) : (
+            <Text sx={sx.unfinishedMessage}>
+              Complete the remaining indicators for this access measure by
+              entering details.
+            </Text>
+          )}
+        </>
+      );
```

Comment:
Nit: Don't need the fragment wrapper here.

Reply:
@braxex I tried not having them and it doesn't like it. Which ones could I remove?

Reply:
Note: I changed this to make it dumb like you suggested below so maybe this doesn't apply anymore

Reply:
Yeah don't sweat it.